### DISCUSSION TITLE
ASCII2NC.conf time summary width issue

### DISCUSSION THREAD

**Discussion by hertneky** [dtcenter/METplus#1251] [Configuration] [answered]
I'm using ascii2nc for SURFRAD data in METplus. In the time_summary options, I need ASCII2NC_TIME_SUMMARY_WIDTH = { beg = -21600; end = 0; }, an average over the previous 6h period.

End lines from log are File "/contrib/METplus/METplus-4.0.0/produtil/config.py", line 169, in get_value
    raise KeyError(key)
KeyError: ' beg = -21600; end = 0; '

John HG suspects "that the METplus wrappers are likely assuming that width is always set to an integer and it failing a validation step".

**Comment by georgemccabe** [comment, marked as answer]
Hi @hertneky, John HG is correct that the wrapper assumes width is an integer. I see in the documentation now that it can also be a dictionary. This is an easy fix that I can make so it will be available in the 4.1.0-beta4 release next week.

**Reply by georgemccabe** [reply]
I created a GitHub issue to track this change: #1252

**Reply by hertneky** [reply]
sounds good - thanks George
